handbook/engineering/eng_org.md

```diff
@@ -38,28 +38,28 @@ Plans can change!
 
 - [Beyang Liu](../../../company/team/index.md#beyang-liu), CTO
 - [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him), [VP Engineering](roles.md#vp-engineering)
-    - Search and replace (WIP name) {#search-and-replace}
-        - FQ3 Director of Engineering, Nick acting manager until then.
-        - [Team: Search core backend](search/index.md#search-core-backend-eng)
-        - [Team: Search product](search/index.md#search-product-eng)
-        - Code intelligence
-            - [Team: Code intelligence](code-intelligence/index.md#code-intelligence-eng)
-        - Campaigns
-            - [Team: Campaigns](campaigns/index.md#campaigns-eng)
-    - Web (WIP name) {#web}
-        - [Jean du Plessis](../../../company/team/index.md#jean-du-plessis-he-him) ([Engineering Manager](roles.md#engineering-manager))
-        - [Team: Frontend platform](web/index.md#frontend-platform-eng)
-        - [Team: Extensibility](web/index.md#extensibility-eng)
-        - Code insights
-            - [Team: Code insights](code-insights/index.md#code-insights-eng)
-    - Platform and infrastructure {#platform-and-infrastructure}
-        - [FQ1 Director of Engineering](https://jobs.lever.co/sourcegraph/8fcbaade-e511-4d93-bd9b-9cc7ec3438af), Nick acting manager until then.
-        - Security
-            - [Team: Security](security/index.md#security-eng)
-        - Distribution
-            - [Team: Distribution](distribution/index.md#distribution-eng)
-        - [Team: Core application](cloud/index.md#core-application-eng)
-        - [Team: Backend platform](cloud/index.md#backend-platform-eng)
+  - Search and replace (WIP name) {#search-and-replace}
```

Comment:
please verify this still renders correctly locally (I was having issues)

Reply:
LGTM in VS Code preview.

Reply:
Yeah this broke the formatting (engineers are supposed to be nested under the EM)

It is annoying that our renderer has some differences with GitHub and VS code renderer

![Screen Shot 2021-02-10 at 10 20 49 AM](https://user-images.githubusercontent.com/754768/107553527-b49fdf80-6b89-11eb-9cc0-15ed75d98bc3.png)

Reply:
🤦🏼, I'll push a fix. How do you preview with our rendering?

Reply:
https://github.com/sourcegraph/about/commit/dc5a598bf31401130234bb4d13f720df87b9fc30

Reply:
`make serve` is how you can test locally. Your fix didn't seem to do the trick but I fixed it here and added a comment for the future: 457c355d4297f75e357e72087dc85f40929af39a